recipes/Dessert/frozen_chocolate_cake.md

```diff
@@ -0,0 +1,38 @@
+# Frozen Chocolate Cake
+(not frikin pudding)
+
+![Difficulty Badge](https://img.shields.io/badge/Difficulty-10%25-brightgreen.svg)
+![Speed Badge](https://img.shields.io/badge/Speed-10min-brightgreen.svg)
+![Reception Badge](https://img.shields.io/badge/Reception-Very%20Positive-brightgreen.svg)
+![Cost Badge](https://img.shields.io/badge/Cost-Very%20Cheap-brightgreen.svg)
+![Vegetarian Friendly Badge](https://img.shields.io/badge/Vegetarian-True-brightgreen.svg)
+![Vegan Friendly Badge](https://img.shields.io/badge/Vegan-False-red.svg)
+
++ **First cooked by:**
+[@Suuuuha](https://github.com/suuuuha)
+[@Karyum](https://github.com/karyum)
+
++ **Date:** Week 3 FACN2
++ **Author of Recipe:**
+[10min website in hebrew](http://www.10dakot.co.il/%D7%A2%D7%95%D7%92%D7%AA-%D7%9B%D7%93%D7%95%D7%A8%D7%99-%D7%A9%D7%95%D7%A7%D7%95%D7%9C%D7%93/)
+
+### Ingredients
+- 400 grams of biscuits.
+- 250 ml cream
+- 100 grams bitter chocolate bar
+#### Frosting
+- 125 ml cream
+- 100 grams bitter chocolate bar.
+
+### Preparation
+
+1. Smash the biscuits into really small crumbs.
+2. Take a bowl and add the chocolate (1 bar) with the cream (250 ml).
+3. Heat them up above a pan of boiling water, keep mixing until you get smooth mixture.
+4. Add the biscuit crumbs to the smooth mixture and keep mixing until all the crumbs are covered with chocolate.
+5. Pour all the mixture into an english cake tray and then press firmly so it would stick to the bottom of the tray.
```

Comment:
I choose to believe this one
![](https://ae01.alicdn.com/kf/HTB13eXKLXXXXXXGaXXXq6xXFXXXx/-new-18-15cm-font-b-cake-b-font-mold-26-font-b-English-b-font.jpg)